sql-cli/examples/_dags/simple.py

```diff
@@ -0,0 +1,96 @@
+# NOTE: This is an auto-generated file. Please do not edit this file manually.
```

Comment:
It would be great to have tests running validating the DAG Generator works as expected. I wonder if the content of the `examples` directory should be within tests. We could have an integration test that inputs SQL files and outputs the DAG file and checks if the DAG file is what we want it to be.